extensions/wikia/CommunityPage/models/CommunityPageSpecialCommunityTodoListModel.class.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+class CommunityPageSpecialCommunityTodoListModel {
+	public function getData() {
+		$title = Title::newFromText(
+			wfMessage( 'communitypage-todo-module-page-name' )->inContentLanguage()->plain(),
```

Comment:
I'd just do something like:
```php
$messageKey = 'community-to-do-list';
$message = wfMessage( $messageKey );
$title = Title::newFromText( $messageKey, NS_MEDIAWIKI );
$data = [
	'haveContent' => false,
	'editUrl' => $title->getFullURL( ['action' => 'edit'] ),
];

if ( !$message->isDisabled() ) {
	$data['haveContent'] = true;
	$data['data'] = $message->parse();
}

return $data;
```

The message key could also be a constant on the class if it might need to be checked outside this method. What do you think?